parser/rdjsonl.go

```diff
@@ -0,0 +1,35 @@
+package parser
+
+import (
+	"bufio"
+	"io"
+
+	"github.com/reviewdog/reviewdog/proto/rdf"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+var _ Parser = &RDJSONLParser{}
+
+// RDJSONLParser is parser for rdjsonl format.
+type RDJSONLParser struct{}
+
+func NewRDJSONLParser() *RDJSONLParser {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function NewRDJSONLParser should have comment or be unexported